designer/client/outputs/outputs-edit.tsx

```diff
@@ -42,7 +40,7 @@ class OutputsEdit extends React.Component<Props, State> {
   };
 
   render() {
-    const { data } = this.props;
+    const data = this.context;
```

Comment:
for class components, you need to change the constructor to accept two params 
https://github.com/facebook/react/issues/13944#issuecomment-432858365